scene/gui/tab_bar.h

```diff
@@ -172,7 +177,7 @@ class TabBar : public Control {
 	void _on_mouse_exited();
 
 	void _shape(int p_tab);
-	void _draw_tab(Ref<StyleBox> &p_tab_style, Color &p_font_color, int p_index, float p_x, bool p_focus);
+	void _draw_tab(Ref<StyleBox> &p_tab_style, Color &p_font_color, Color &p_icon_color, int p_index, float p_x, bool p_focus);
```

Comment:
```suggestion
	void _draw_tab(Ref<StyleBox> &p_tab_style, Color &p_font_color, const Color &p_icon_color, int p_index, float p_x, bool p_focus);
```
The `p_font_color` case is wrong but let's fix this one